packages/engine/Source/Core/Math.js

```diff
@@ -183,6 +183,13 @@ CesiumMath.SOLAR_RADIUS = 6.955e8;
  */
 CesiumMath.LUNAR_RADIUS = 1737400.0;
 
+/**
+ * 256
+ * @type {Number}
+ * @constant
+ */
+CesiumMath.TWO_FIFTY_SIX_BYTES = 256;
```

Comment:
Keep in mind that `IndexDatatype.UNSIGNED_BYTE` corresponds to just one byte, and one byte = 256 bits.
```suggestion
CesiumMath.TWO_FIFTY_SIX_BITS = 256;
```